DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/_ActivityTable.cshtml

```diff
@@ -1,40 +1,45 @@
 ﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Reports
-
 @model UsageStatsTableViewModel
 
 <table role="table" class="nhsuk-table-responsive">
   <thead role="rowgroup" class="nhsuk-table__head">
-    <tr role="row">
-      <th role="columnheader" scope="col">
-        Period
-      </th>
-      <th role="columnheader" scope="col">
-        Registrations
-      </th>
-      <th role="columnheader" scope="col">
-        Completions
-      </th>
-      <th role="columnheader" scope="col">
-        Evaluations
-      </th>
-    </tr>
+  <tr role="row">
+    <th role="columnheader" scope="col">
+      Period
+    </th>
+    <th role="columnheader" scope="col">
+      Registrations
+    </th>
+    <th role="columnheader" scope="col">
+      Completions
+    </th>
+    <th role="columnheader" scope="col">
+      Evaluations
+    </th>
+  </tr>

```

Comment:
Is there a reason for the change in indentation here? Looks like <tr> is now in line with <thead>